cmd/draft/draft.go

```diff
@@ -117,13 +128,42 @@ func teardown() {
 
 func ensureDraftClient(client *draft.Client) *draft.Client {
 	if client == nil {
-		return draft.NewClient(&draft.ClientConfig{
-			ServerAddr: draftHost,
-			Stdout:     os.Stdout,
-			Stderr:     os.Stderr,
-		})
+		return client
 	}
-	return client
+	return newClient()
+}
+
+func newClient() *draft.Client {
+	cfg := &draft.ClientConfig{ServerAddr: draftHost, Stdout: os.Stdout, Stderr: os.Stderr}
+	if tlsVerify || tlsEnable {
+		if tlsCaCertFile == "" {
+			tlsCaCertFile = os.ExpandEnv(tlsCaCertDefault)
```

Comment:
Have you taken a look at https://github.com/kubernetes/helm/pull/3224? Seems like we'll need to port that patch over here.

Reply:
I changed the checks to compare against the default values for each tls flag value. Empirically  verified, but curious about your thoughts since you addressed this in helm recently.